SYCL/Basic/kernel_bundle/kernel_bundle_api.cpp

```diff
@@ -233,8 +233,6 @@ int main() {
                                                                 {Kernel3ID});
     // 3 SPIRV images - 3 calls to piextDeviceSelectBinary are expected
```

Comment:
This comment should probably be removed too.

Reply:
Right, thanks.